onboarding/campus-access.md

```diff
@@ -38,3 +38,13 @@ In addition to the above requirement, please also complete the following steps s
 * Check-in with your supervisor if you have any plans to come on campus, as you will need to be authorized first.
 * Read over the [NeuroPoly Lab Reopening guide](https://docs.google.com/presentation/d/1JtJLcgdA7aXuxRHIzu1hN_lLjYK8PI0BKa0OK-155TE/edit#slide=id.p1) to learn about the mandatory training procedures that are necessary for returning to the lab.
 * Check the pinned messages in the [\#covid-19 ](https://app.slack.com/client/T034UD4QN/C015YFVFWJC) channel on NeuroPoly's Slack for the most up-to-date info, as the situation is likely to change as the pandemic evolves.
+
+### WiFi Access
+
+#### (Non-eduroam Campus WiFi)
+
+TODO. (JN: As a non-Montreal employee, I don't know anything about how non-eduroam campus WiFi works.)
+
+#### Eduroam
+
+As part of your PolyMTL CAS account (`@polymtl.ca`), you are granted access to the 'eduroam' network across campuses internationally. Setting up a connection to this network requires additional steps beyond just logging in. For more information on how to log into eduroam, please visit the ['Computing Resources - Eduroam'](https://intranet.neuro.polymtl.ca/computing-resources/eduroam.html) page.
```

Comment:
Also, I noticed that elsewhere in this file, we're linking directly to the rendered .html links, so I copied that convention here too.

@kousu does the website's current form of markup allow for anchors/references that we can link to instead, like Sphinx does? I'm not sure if that's something we want to aim for, e.g. to avoid dead links.

Reply:
This is a sphinx doc, so it is hyperlink aware. I tested to double-check my memory and this syntax works fine:

```suggestion
### WiFi Access

#### (Non-eduroam Campus WiFi)

TODO. (JN: As a non-Montreal employee, I don't know anything about how non-eduroam campus WiFi works.)

#### Eduroam

As part of your PolyMTL CAS account (`@polymtl.ca`), you are granted access to the 'eduroam' network across campuses internationally. Setting up a connection to this network requires additional steps beyond just logging in. For more information on how to log into eduroam, please visit the ['Computing Resources - Eduroam'](/computing-resources/eduroam.md) page.
```

MyST makes sure that if the link works on GitHub it will work when rendered by sphinx.

Reply:
Actually that link won't work on GitHub? But it does work in sphinx, sphinx rewrites it to a relative link:

```
<a class="reference internal" href="../computing-resources/eduroam.html"><span class="doc std std-doc">‘Computing Resources - Eduroam’</span></a>
```

Reply:
Even though this gets rid of the `.html`, it's still dependent on the exact location of the page. (E.g. page reorgs/renames will still break this kind of reference.)

I think I was looking for, like... links to sections that aren't dependent on exact page paths? Like `:ref:'section-reference'`.

It's not a huge deal though, I think?